website/docs/d/private_link_service.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: ""
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service"
+sidebar_current: "docs-azurerm-datasource-private-link-service"
+description: |-
+  Use this data source to access information about an existing Private Link Service.
+---
+
+# Data Source: azurerm_private_link_service
+
+Use this data source to access information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service" "example" {
+  name                = "myPrivateLinkService"
+  resource_group_name = "PrivateLinkServiceRG"
+}
+
+output "private_link_service_id" {
+  value = "${data.azurerm_private_link_service.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link service.
+
+* `resource_group_name` - (Required) The name of the resource group in which the private link service resides. Changing this forces a new resource to be created.
```

Comment:
Because this is a datasource:
```suggestion
* `resource_group_name` - (Required) The name of the resource group in which the private link service resides.
```